public/app/features/search/components/SearchItem.tsx

```diff
@@ -48,15 +44,16 @@ export const SearchItem: FC<Props> = ({ item, editable, onToggleSelection, onTag
     <li
       aria-label={selectors.dashboards(item.title)}
       className={cx(styles.wrapper, { [styles.selected]: item.selected })}
-      onClick={navigate}
     >
       <SearchCheckbox editable={editable} checked={item.checked} onClick={toggleItem} />
-      <Icon className={styles.icon} name="th-large" />
-      <div className={styles.body} onClick={onItemClick}>
-        <span>{item.title}</span>
-        <span className={styles.folderTitle}>{item.folderTitle}</span>
-      </div>
-      <TagList tags={item.tags} onClick={tagSelected} className={styles.tags} />
+      <a href={item.url} className={styles.link}>
+        <Icon className={styles.icon} name="th-large" />
+        <div className={styles.body} onClick={onItemClick}>
+          <span>{item.title}</span>
+          <span className={styles.folderTitle}>{item.folderTitle}</span>
+        </div>
+        <TagList tags={item.tags} onClick={tagSelected} className={styles.tags} />
```

Comment:
Clicking on tags no longer works, they need a e.stopPropagation() or e.preventDefault to stop the event from bubbling to the anchor

Reply:
It's there in the `tagSelected`, but it doesn't work. Neither does `preventDefault` nor their `event.nativeEvent` alternatives. Could be something to do with angular routing and how it attaches listeners to anchor tags. 

The only solution I was able to come up with is to attach an event listener to the tags' `ref` and prevent default there. Can't think of any better alternatives.